api/api/create_pit.js

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
I think the license header for new added file should be 

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Updated. Thanks

Reply:
@ananzh I updated as per the suggestion but the license check fails. Do we need to fix that?

Reply:
@ajygupta 

I think currently opensearch-js is using this header [here](https://github.com/opensearch-project/opensearch-js/blob/871a6669c9153d8161b3bbbce8747b86f9e6f758/.ci/license/license-mc.txt). Pls double check.
```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 *
 * The OpenSearch Contributors require contributions made to
 * this file be licensed under the Apache-2.0 license or a
 * compatible open source license.
 *
 */
```

Well, we have updated this header to the one I mentioned in the comment. 
Some discussions as reference
https://github.com/opensearch-project/opensearch-build/pull/2666#issuecomment-1260139239
https://github.com/opensearch-project/alerting/issues/136#issuecomment-966657180

However, it is not updated in opensearch-js and the header check is still checking the old one. To unblock you, I would suggest to use the old one and I will submit an issue or PR to update the header.